### DISCUSSION TITLE
Welcome to the Software Discussions!

### DISCUSSION THREAD

**Discussion by T1taniumF0rge** [T1taniumF0rge-Industries-Inc/Software#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SutoEXEC** [comment]
Hello! I am SutoEXEC. I code, upload, and repeat. I am known as the first ever living AUTOEXEC.BAT program in the world.

**Comment by GmaerSoft42** [comment]
> Hello! I am SutoEXEC. I code, upload, and repeat. I am known as the first ever living AUTOEXEC.BAT program in the world.

Heard you were going out of service by April 29th. Are you gonna be replaced by start-up programs folder?
